### PR TITLE
Followup to #621

### PR DESCRIPTION
**Pull request by simonschaufi**
- Truncate strings in charts
- move label box a bit more to the left to avoid overlapping

### CODE REVIEW

**Comment by kevinpapst** [comment]
30px right for the label box moves it almost into the chart (chrome on mac). doesn't look good here, because there is a lot of space left between the chart and the box border. I guess its highly depending on usernames and translation of the label ... any idea? JS positioning?

image

Truncate is a good idea indeed.

**Comment by simonschaufi** [comment]
try it with long usernames like simonschaufi. the original position then overlapped

File: extensions/ki_budget/js/budget_func.js [hunk 1]
```diff
@@ -72,133 +72,136 @@ function recalculate_budget_window() {
  */
 function budget_extension_set_heightTop() {
 	budget_extension_get_dimensions();
-    if (!extensionShrinkMode) {
-        $("#budgetArea").css("height", budget_h);
-    } else {
-        $("#budgetArea").css("height", "20px");
-    }
-    
-    export_extension_set_TableWidths();
```
Comment (simonschaufi) [inline extensions/ki_budget/js/budget_func.js:81]: I didnt understand why this was there. maybe copy paste error, maybe mistake...

Comment (kevinpapst) [inline extensions/ki_budget/js/budget_func.js:81]: Probably, or likely Copy&Paste. But I have no clue either. I work the same way all the time: change and see what happens. Probably we should set the label box to a fixed with and truncate the username?

**Comment by kevinpapst** [comment]
Hey, we can merge that, I was just wondering if you are looking into another option? Probably javascript positioning, like the rest of the UI

**Comment by simonschaufi** [comment]
let's make another PR for that...

**Pull request merged**